FAQ.md

```diff
@@ -72,6 +72,55 @@ Default 60 seconds, can be changed using `metrics-resolution` flag. We are not r
 
 ## Known issues
 
+#### Unable to authenticate the request due to an error: x509: certificate signed by unknown authority
+
```

Comment:
```suggestion
#### Incorrectly configured front-proxy certificate

```
Metrics Server needs to validate requests coming from kube-apiserver. You can recognize problems with front-proxy certificates if you observe line below in your logs: